service/application/src/main/java/com/solace/maas/ep/common/messages/ScanLogsMessage.java

```diff
@@ -0,0 +1,29 @@
+package com.solace.maas.ep.common.messages;
+
+import com.solace.maas.ep.runtime.agent.plugin.mop.MOPMessage;
+import com.solace.maas.ep.runtime.agent.plugin.mop.MOPMessageType;
+import com.solace.maas.ep.runtime.agent.plugin.mop.MOPProtocol;
+import com.solace.maas.ep.runtime.agent.plugin.mop.MOPUHFlag;
+
+public class ScanLogsMessage extends MOPMessage {
+    String orgId;
+
+    String scanId;
+
+    String log;
+
+    String timestamp;
```

Comment:
What is the format of this timestamp? Is it in Epoch Milliseconds? If yes, can we change the type to `Long` instead?

Reply:
The timestamp key that was part of the json object was of type "Instant" converted to a String. However, after removing the json object, the timestamp is now parsed from the log event, which is of type long.